notte/env.py

```diff
@@ -120,20 +120,13 @@ def disable_auto_scrape(self) -> "NotteEnvConfig":
         self.auto_scrape = False
         return self
 
-    @staticmethod
-    def use_llm() -> "NotteEnvConfig":
-        return NotteEnvConfig().llm_data_extract().llm_action_tagging()
-
-    @staticmethod
-    def disable_llm() -> "NotteEnvConfig":
-        return (
-            NotteEnvConfig(
-                scraping=ScrapingConfig(type=ScrapingType.SIMPLE),
-                action=MainActionSpaceConfig(type=ActionSpaceType.SIMPLE),
-            )
-            .dom()
-            .disable_auto_scrape()
-        )
+    def use_llm(self) -> "NotteEnvConfig":
+        return self.llm_data_extract().llm_action_tagging()
+
+    def disable_llm(self) -> "NotteEnvConfig":
+        self.scraping = ScrapingConfig(type=ScrapingType.SIMPLE)
```

Comment:
Otherwise the whole scraping config is updated
```suggestion
        self.scraping = ScrapingConfig(type=ScrapingType.SIMPLE)
```